src/maintainer/knowledge_base.rst

```diff
@@ -1136,3 +1136,117 @@ key with ``docker_image``. Also ``cdt_name`` ensures the CDTs match the CentOS
+requirements automatically.
+
+.. note::
+
+  **How is CUDA provided at the system level?**
+
+  * On Linux, Nvidia provides official Docker images, which we then
+    `adapt <https://github.com/conda-forge/docker-images>`_ to the Conda Forge needs.
+
+  * On Windows, the compilers need to be installed for every CI run. This is done through the
+    `conda-forge-ci-setup <https://github.com/conda-forge/conda-forge-ci-setup-feedstock/>`_ scripts.
+    Do note that the Nvidia executable won't install the drivers because no GPU is present in the machine.
+
+  **How is ``cudatoolkit`` selected at install time?**
+
+  Conda exposes the maximum CUDA version supported by the installed Nvidia drivers through a virtual package
+  named ``__cuda``. By default, ``conda`` will install the highest version available
+  for the packages involved. However, prior to v4.8.4, ``__cuda`` versions would not be part of the
+  constraints, so you would always get the latest one, regardless the supported CUDA version.
+
+  If for some reason you want to install a specific version, you can use::
+
+    conda install your-gpu-package cudatoolkit==10.1
+
+Testing the packages
+--------------------
+
+Since the CI machines do not feature a GPU, you won't be able to test the built packages as part
+of the conda recipe. For now, the workaround is to enable the Azure artifacts for your feedstock
+(see :ref:`azure-config`) and then perform the tests locally.
+
+Common problems and known issues
+--------------------------------
+
+Helping your build system locate CUDA
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some build systems might need some help finding CUDA, so you might need to pass the value
+of ``CUDA_HOME`` as some kind of flag. Some examples include:
+
+* Old-style CMake (using the deprecated ``FindCUDA`` module), you might need to pass this flag::
+
+    -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME}
+
+* On Windows, if you need to pass ``CUDA_PATH``, CMake might choke on a backslash. In that case,
+  you can overwrite ``CUDA_PATH`` as follows::
+
+    set "CUDA_PATH=%CUDA_PATH:\=/%"
+
+* On Windows packages that are built with MSBuild, new-style CMake (using ``FindCUDAToolkit`` module)
+  you might need to define this environment variable::
+
+    set "CudaToolkitDir=%CUDA_PATH%"
+
+
+CUDA 11 and CDT packages
+^^^^^^^^^^^^^^^^^^^^^^^^
+If your package requires both CUDA and some CDTs on Linux, you will need to patch the
+``.ci_support/*.yml`` files corresponding to CUDA 11 and above so they reflect
+``cdt_name: cos7``. Do note that these changes do not survive feedstock rerenders!
+
+This should be fixed at some point, but for now you need to do it manually. Apologies!
+
+Submitting CUDA-enabled packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Right now, ``staged-recipes`` cannot build CUDA-enabled packages. You will have to
+start with a simpler version (CPU only) and then add the CUDA bits once you
+have obtained a feedstock.
```

Comment:
@isuruf Didn't you just add cuda support to staged recipes?

Reply:
Oh, I didn't know about that! I guess it's [this commit](https://github.com/conda-forge/staged-recipes/commit/d21496744b5a677c1d9ca42328a76787ca37e426)?

So basically, since it detects CUDA automatically, we don't have to do anything special and can just remove this section, right?

Reply:
Yes I think so!

Reply:
Yep, just checked here https://github.com/conda-forge/staged-recipes/pull/13486. I removed this section now.